src/widgets/list/content-view.js

```diff
@@ -34,6 +42,11 @@ module.exports = WidgetContentView.extend({
     return this;
   },
 
+  _initBinds: function () {
+    this.listenTo(this.model, 'change:title', this.render);
+    this.listenTo(this._dataviewModel, 'change:data', this.render);
```

Comment:
Fixed in d4a2a22 (comment do still appear since the fix was on the next line :()